hydropandas/io/bro.py

```diff
@@ -657,6 +844,16 @@ class of the observations, e.g. GroundwaterObs or WaterlvlObs
         by default you get a prompt if you want to download over a 1000
         observations at once. if ignore_max_obs is True you won't get the
         prompt. The default is False
+    engine : str, optional
+        Select how data from the bro-database is obtained, options are 'hydropandas',
+        'brodata' or 'brodata_gm'. When engine='brodata_gm' use the dataset
+        Grondwatermonitoring (GM) in samenhang - karakteristieken, hosted by PDOK. This
+        up-to-date dataset combines well- and tube-properties. So users do not have to
+        download each individual Groundwater Monitoring Well (GMW), which speeds up the
+        request. The gm-dataset does not contain the attributes `tube_top` and
+        `ground_level`, so you need to use engine='brodata' or 'hydropandas' if you
+        need those. The Groundwater Level Dossiers (GLD) are still downloaded
+        individually. The default is True. The default is 'hydropandas'.
```

Comment:
What is the default?

```suggestion
        individually. The default is 'hydropandas'.
```